docs/webpack.md

```diff
@@ -0,0 +1,100 @@
+# Webpack usage examples
+
+## Setup
+
+### Javascript
+
+Create `src/index.js`:
+```ts
+const yargs = require('yargs')
+
+console.log(yargs.parse())
+```
+
+Create `webpack.config.js`:
+```js
+const path = require('path');
+const nodeExternals = require('webpack-node-externals');
+
+module.exports = {
+  entry: './src/index.js',
+  externals: [nodeExternals()],
```

Comment:
Removing all node_modules from the pack just to get rid of yargs webpack issues is excessive.

I will update this next week.

Reply:
Fixed